airflow/providers/databricks/sensors/databricks_partition.py

```diff
@@ -182,7 +182,11 @@ def _generate_partition_query(
         partition_columns = self._sql_sensor(f"DESCRIBE DETAIL {table_name}")[0][7]
         self.log.debug("Partition columns: %s", partition_columns)
         if len(partition_columns) < 1:
-            raise AirflowException(f"Table {table_name} does not have partitions")
+            message = f"Table {table_name} does not have partitions"
```

Comment:
We'll need to add the TODO like https://github.com/apache/airflow/blob/659d94f0ae89f47a7d4b95d6c19ab7f87bd3a60f/airflow/providers/airbyte/sensors/airbyte.py#L63 so that it's less likely for us to forget it

Reply:
Made the required change.